### DISCUSSION TITLE
use hyphens in named interpolation in json file

### DISCUSSION THREAD

**Discussion by acastillo9** [intlify/vue-i18n#1715] [Q&A]
there is some workaround to use hyphens in named interpolation, for instance the key max-consecutive-length?

example:

`{
  "CONTINUOUS_DUPLICATE_CHARACTERS": "Exceeding {max-consecutive-length} identical characters in a row is not allowed"
}`

**Comment by ChrisGV04** [comment]
I don't think so, since variable names in Javascript cannot include hyphens, but I'm not sure.

Is it possible that you use camel case `maxConsecutiveLenght` instead?
